frontend-vue-js/.env

```diff
@@ -1,2 +1,4 @@
-VUE_APP_APIUSERSURL=http://localhost:8081/api/users
-BACKEND_APP_URL = http://localhost:8081/
\ No newline at end of file
+VUE_APP_USER_API_ENDPOINT = api/users
+
+VUE_APP_LOGIN_ENDPOINT = auth/login/
+VUE_APP_REGISTRATION_ENDPOINT = auth/signup/
```

Comment:
Will we put all endpoints here? And just use them in app? If yes, then we need to inform rest of the our team.

Reply:
Don't worry baby 😎

Reply:
It is same aproach. We use .env conf file before repacking

Reply:
I now, but we did not make a agreement about what we would put here. But if we will put all endpoints, it is nice for me.